database-commons/src/main/java/co/cask/DBManager.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright © 2015-2018 Cask Data, Inc.
```

Comment:
please update all copyrights to 2019 since these are now new files.

Reply:
done